### PR TITLE
Update task ux

### PR DESCRIPTION
**Pull request by Thrillberg**
# 🔗 Issue

# ✍️ Description

# 📷 Screenshots/Demos

### CODE REVIEW

File: app/views/organizations/tasks/_form.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<%= form_for [@pet, @task], remote: true do |f| %>
```
Comment (edwinthinks) [inline app/views/organizations/tasks/_form.html.erb:1]: Somewhat of a unrelated thing I noticed is that maybe the `tasks` folder should be under `pets`. I think we got confused (or I did) when I didn't realize that we have a `pets/tabs` section instead.

File: app/views/organizations/tasks/edit.html.erb [hunk 2]
```diff
@@ -1,26 +1,5 @@
...
-
-    <% end %>
-  </turbo-frame>
-</div>
\ No newline at end of file
+<%= turbo_frame_tag @task do %>
+  <%= bootstrap_form_with model: @task, url: pet_task_path(@task.pet) do |f| %>
+    <%= render partial: 'form', locals: {f: f} %>
+  <% end %>
+<% end %>
```
Comment (edwinthinks) [inline app/views/organizations/tasks/edit.html.erb:5]: Nice!

**Review by edwinthinks — APPROVED** [review]
Minor thoughts! And we can get this merged in imo!

P.S - I think the next step is to add the checkbox to mark tasks as done or not done. Could you create a issue for that?

File: app/views/organizations/tasks/_form.html.erb (hunk 1 above)
Comment (Thrillberg) [reply app/views/organizations/tasks/_form.html.erb:1]: I've moved some templates in the latest commit but I'm not sure if it's an improvement? Things are still kind of split up in multiple places. What do you think?

**Comment by Thrillberg** [comment]
Issue created here: https://github.com/rubyforgood/pet-rescue/issues/386

**Comment by Thrillberg** [comment]
I've removed the failing test, which is not a great solution. But that test seems mostly meaningless to me, at this point, because on validation error the controller doesn't actually redirect or fail or anything bad--it just gives a 200OK. Is that the desired pattern, @edwinthinks ?

**Comment by edwinthinks** [comment]
@Thrillberg I think that is a fine solution for now. I think adding more meaningful tests will come later.

**Pull request merged**